apps/frontend/src/components/EditableTable.tsx

```diff
@@ -0,0 +1,64 @@
+import { Table } from 'antd';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
+import { useEffect, useState } from 'react';
+import { getAllTransactions } from '../features/transactions/transactionSlice';
+import { auth } from '../firebase';
+import { User } from 'firebase/auth';
+
+const EditableTable = () => {
+  const dispatch = useAppDispatch();
+  const { user, isLoading: isUserLoading } = useAppSelector(
+    (state) => state.user
+  );
+  const firebaseUser = auth.currentUser as User;
+  useEffect(() => {
+    if (user) {
+      dispatch(getAllTransactions(firebaseUser));
+    }
+    setLoading(false);
+  }, []);
+  const { transactions, isLoading: isTransactionLoading } = useAppSelector(
+    (state) => state.transaction
+  );
```

Comment:
Please look for a way to make axios always get Authorization header automatically. Without this we will need to duplicate this code in each place, where we do a request to backend.

What I don't like here is that we need to pass "firebaseUser" in every dispatch.

maybe we can import auth.currentUser even in redux slices or I don't know.

Try to find a way to make it easier :) 
If nothing we can do - also OK

Reply:
created an axios instance with a request interceptor which automatically adds an authorization token to each request